test/e2e/e2ethanos/services.go

```diff
@@ -25,6 +27,11 @@ import (
 
 const logLevel = "info"
 
+var defaultBackoffConfig = util.BackoffConfig{
+	MinBackoff: 300 * time.Millisecond,
+	MaxBackoff: 30 * time.Second,
```

Comment:
kind of long...

Reply:
But if that works for you now, let's stick to this. The problem with this is that waiting for a legitimate lack of proper metric  value will take ages.